benchmarking/README.md

```diff
@@ -88,60 +93,128 @@ git clone https://github.com/sambanova/ai-starter-kit.git
+
+### Deploy the starter kit GUI
 
 Ensure you are in the `benchmarking` folder and run the following command:
-```
+
+```shell
 streamlit run streamlit/app.py --browser.gatherUsageStats false 
 ```
 
 After deploying the starter kit, you will see the following user interface:
 
 ![perf_eval_image](./imgs/performance_eval.png)
 
-## Use the starter kit 
+### Quickstart 
 
 After you've deployed the GUI, you can use the starter kit. More details will come in the following sections, however the general usage is described in the comming bullets: 
 
-1. In the left side bar, select one of the two app functionalities: `Performance evaluation` or `Performance on chat`.
+1. In the left side bar, select one of the three app functionalities (Click on each section to go to the full details): 
+  - [`Synthetic Performance Evaluation`](#synthetic-performance-evaluation): Evaluate the performance of the selected LLM on data generated by this benchmarking tool.
+  - [`Custom Performance Evaluation`](#custom-performance-evaluation): Evaluate the performance of the selected LLM on custom data specified by you.
+  - [`Performance on Chat`](#performance-on-chat): Evaluate the performance of the selected LLM in a chat interface.
 
-2. If the LLM deployed is a Composition of Experts, introduce the LLM expert and configure each of the parameters related to each functionality. Otherwise, just do the latter.
+2. If the deployed LLM is a Composition of Experts (CoE), specify the desired expert in the corresponding text box, then set the configuration paramters. If the deployed LLM is not a CoE, simply set the configuration parameters.
 
-3. Press the `Run` button, wait and analyze results in the middle of the screen. In the case of `Performance on chat` functionality, users are able to interact with the LLM in a chat interface.  
+3. After pressing the `Run` button, the program will perform inference on the data and product results in the middle of the screen. In the case of `Performance on Chat` functionality, users are able to interact with the LLM in a multi-turn chat interface.  
 
-### Performance evaluation workflow
+### Full Walkthrough
 
-There are two options that users can choose from. The first one is running the performance evaluation process using the terminal, while the other is using the Streamlit app.
+There are 3 options on the left side bar for running the benchmarking tool. Pick the walkthrough that best suits your needs.
 
-#### Using streamlit app
+#### Synthetic Performance Evaluation
 
-Choose the option `Performance evaluation` on the left side bar, the following interface shows up: 
+This option allows you to evaluate the performance of the selected LLM on synthetic data generated by this benchmarking tool.
 
-![perf_eval_image](./imgs/performance_eval.png)
+1. Enter a model name
+
+  - If the model specified in the `ai_starter-kit/.env` file is not a CoE model, you can skip this step.
```

Comment:
hmm I know this technically works, but in file output names the name of the model will be missing, so I think we should still require the model name for now.

Reply:
Good call. I added in another bullet to be more explicit about the standalone models. ✅